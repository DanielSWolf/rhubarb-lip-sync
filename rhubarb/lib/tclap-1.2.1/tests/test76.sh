#!/bin/sh

# failure  validates that the correct error message
# is displayed for XOR'd args
../examples/test20 -ba > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test76.out; then
	exit 0
else
	exit 1
fi

