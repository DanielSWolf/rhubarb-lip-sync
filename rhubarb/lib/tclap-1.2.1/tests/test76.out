PARSE ERROR: Argument: -b (--btmc)
             Mutually exclusive argument already set!

Brief USAGE: 
   ../examples/test20  {-a|-b} [--] [--version] [-h]

For complete USAGE and HELP type: 
   ../examples/test20 --help

